onnxruntime/core/common/profiler_common.cc

```diff
@@ -50,4 +50,3 @@ std::string demangle(const std::string& name) {
 
 }  // namespace profiling
 }  // namespace onnxruntime
```

Comment:
It generally better to keep a newline on EOF, maybe run clang-format on all the files? Note, if you use vscode for editing, it is configured to be auto handle the whitespaces, see `.vscode/settings.json`

Reply:
I agree! Not sure what happened here, I see a newline locally, hopefully the clang-format will fix this.

Reply:
Oh, I think there was an additional empty line at the end, and that got removed?